kr_mesh_visualization/README.md

```diff
@@ -0,0 +1,40 @@
+kr_mesh_visualization
+=============
+
+Package to visualize the drone in RViz.
+
+## Message types supported:
+
+Three message types are supported in this package in order to subscribe to the pose data and visualize the drone. The default message type that is being subscribed to is `nav_msgs/msg/Odometry`. The other message types supported are `geometry_msgs/msg/PoseStamped` and `geometry_msgs/msg/PoseWithCovarianceStamped`. The subscribers for the other two message types have been commented out since ROS2 Humble does not allow multiple subscribers to subscribe to a topic with different message types. 
+
+If you need to use another a particular message type, please uncomment the respective subscriber and comment out the rest.
```

Comment:
Change to:
 you will need to replicate the subscriber for the message type you need

Reply:
I have already made the subscriber for the other message types and commented them out in the file